internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_events.go

```diff
@@ -38,8 +37,8 @@ func (e *Events) Watch(f runtime.WatchFunc) {
 	ch := e.add()
 
 	go func() {
-		defer e.delete(ch)
 		defer close(ch)
+		defer e.delete(ch)
```

Comment:
send to closed channel panics, so we should first delete channel from subscribers, then close it